planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/config/crosswalk.param.yaml

```
@@ -29,7 +29,8 @@
 
       # params to prevent stopping on crosswalks due to another vehicle ahead
       obstruction_prevention:
-        enable_obstruction_prevention: false # [-] flag to enable obstruction prevention checking for crosswalk which is in intersection
+        enable_obstruction_prevention: true # [-] flag to enable obstruction prevention feature
```

```suggestion
        enable_obstruction_prevention: false # [-] flag to enable obstruction prevention feature
```
Please keep the default setting